vendor/src/github.com/seccomp/libseccomp-golang/seccomp.go

```diff
@@ -20,7 +20,7 @@ import (
 
 // C wrapping code
 
-// #cgo LDFLAGS: -lseccomp
+// #cgo pkg-config: libseccomp
```

Comment:
This seams to break static linking with "hack/make.sh gccgo" (at least on s390x):

```
/tmp/go-build082546422/github.com/seccomp/liblibseccomp-golang.a(seccomp.cgo2.o): In function `_cgo_68447050d42b_Cfunc_seccomp_arch_add':
/go/src/github.com/docker/docker/vendor/src/github.com/seccomp/libseccomp-golang/seccomp.go:59: undefined reference to `seccomp_arch_add'
/tmp/go-build082546422/github.com/seccomp/liblibseccomp-golang.a(seccomp.cgo2.o): In function `_cgo_68447050d42b_Cfunc_seccomp_arch_exist':
/go/src/github.com/docker/docker/vendor/src/github.com/seccomp/libseccomp-golang/seccomp.go:65: undefined reference to `seccomp_arch_exist
...
```